TOMP-API.yaml

```diff
@@ -741,6 +741,24 @@ paths:
           required: false
           schema: 
             type: string
+        - name: lon
+          in: query
+          description: the longitude of the search location (WGS84)
+          required: false
+          schema:
+            type: number
+        - name: lat
+          in: query
+          description: the latitude of the search location (WGS84)
+          required: false
+          schema:
+            type: number
+        - name: range
```

Comment:
I'd rather use the name `radius` instead of `range`

Reply:
done. Is also compliant with planning.